Merging syntax sections (#5898)
fishtown-analytics/docs.getdbt.com: website/docs/reference/node-selection/syntax.md

dbt run --select "result:<status>+" state:modified+ --defer --state ./<dbt-artifact-path>
```

### The "source_status" status

> Only supported by v1.1 or newer.

Reviewer: Can we remove this? ^

> The following dbt commands produce `sources.json` artifacts whose results can be referenced in subsequent dbt invocations:  

Reviewer: Being there's only one, can we make this simpler? We could even move it into the previous paragraph. 
```suggestion
The `dbt source freshness` command produces a `sources.json` artifact whose results can be referenced in subsequent dbt invocations. 
```

> - `dbt source freshness`

Reviewer: ```suggestion
```

When a job is selected, dbt Cloud will surface the artifacts from that job's most recent successful run. dbt will then use those artifacts to determine the set of fresh sources. In your job commands, you can signal dbt to run and test only on the fresher sources and their children by including the `source_status:fresher+` argument. This requires both the previous and current states to have the `sources.json` artifact available. Or plainly said, both job states need to run `dbt source freshness`.

> After issuing one of the above commands, you can reference the source freshness results by adding a selector to a subsequent command as follows: 

Reviewer: Seems like this is the only command now?
```suggestion
After issuing the `dbt source freshness` command, you can reference the source freshness results by adding a selector to a subsequent command:
```
